policy/modules/contrib/nvme_stas.te

```diff
@@ -0,0 +1,117 @@
+policy_module(nvme_stas, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type nvme_stas_t;
+type nvme_stas_exec_t;
+init_daemon_domain(nvme_stas_t, nvme_stas_exec_t)
+
+type nvme_stas_cache_t;
+files_type(nvme_stas_cache_t)
+
+type nvme_stas_tmpfs_t;
+files_tmp_file(nvme_stas_tmpfs_t)
+
+type nvme_stas_unit_file_t;
+systemd_unit_file(nvme_stas_unit_file_t)
+
+type nvme_stas_var_run_t;
+files_pid_file(nvme_stas_var_run_t)
+
+########################################
+#
+# stas local policy
+#
+allow nvme_stas_t self:capability { net_admin sys_admin };
+allow nvme_stas_t self:capability2 bpf;
+allow nvme_stas_t self:fifo_file rw_fifo_file_perms;
+allow nvme_stas_t self:netlink_kobject_uevent_socket { bind create getattr setopt };
+allow nvme_stas_t self:process setsched;
+allow nvme_stas_t self:tcp_socket create_stream_socket_perms;
+allow nvme_stas_t self:unix_stream_socket create_stream_socket_perms;
+allow nvme_stas_t self:unix_dgram_socket create_socket_perms;
+
+manage_files_pattern(nvme_stas_t, nvme_stas_tmpfs_t, nvme_stas_tmpfs_t)
+fs_tmpfs_filetrans(nvme_stas_t, nvme_stas_tmpfs_t, file)
+can_exec(nvme_stas_t, nvme_stas_tmpfs_t)
+
+manage_dirs_pattern(nvme_stas_t, nvme_stas_var_run_t, nvme_stas_var_run_t)
+manage_files_pattern(nvme_stas_t, nvme_stas_var_run_t, nvme_stas_var_run_t)
+files_pid_filetrans(nvme_stas_t, nvme_stas_var_run_t, file, "last-known-config.pickle" )
+
+kernel_dgram_send(nvme_stas_t)
+kernel_request_load_module(nvme_stas_t)
+
+corecmd_exec_bin(nvme_stas_t)
+
+dev_read_sysfs(nvme_stas_t)
+domain_use_interactive_fds(nvme_stas_t)
+
+files_getattr_all_files(nvme_stas_t)
+files_read_etc_files(nvme_stas_t)
+
+nvme_stas_dbus_chat(nvme_stas_t)
```

Comment:
Interface from the same module should not be used unless there is a good reason; this time it seems just a simple allow rule is needed.